rpi3bplus/debimage-rpi3.yaml

```diff
@@ -1,111 +0,0 @@
-{{- $firmware_version := or .firmware_version "1.20180919" -}}
-{{- $image := or .image "debian-rpi3bplus.img" -}}
-
-architecture: arm64
-
-actions:
-  # Download all needed proprietary stuff first
-  - action: download
-    description: Download latest version of the firmware from GitHub
-    url: https://github.com/raspberrypi/firmware/archive/{{ $firmware_version }}.tar.gz
-    unpack: true     # Unpack downloaded file
-    name: firmware # name for usage directory with unpacked content in other actions
-
-  - action: debootstrap
-    suite: buster
-    components:
-      - main
-    mirror: https://deb.debian.org/debian
-    variant: minbase
-
-  - action: apt
-    description: Install extra packages
-    packages:
-      - sudo
-      - openssh-server
-      - adduser
-      - systemd-sysv
-      - linux-image-arm64
-      - u-boot-rpi
-      - u-boot-menu
-      - wget
```

Comment:
wget was likely added because of what you dropped in 1a490b3de55f8 so maybe wget should also be dropped?